diff --git a/src/packages/fff/fff-wireless/files/etc/uci-defaults/24a-fff-wireless b/src/packages/fff/fff-wireless/files/etc/uci-defaults/24a-fff-wireless
index bc1bff6..54aaf4c 100644
--- a/src/packages/fff/fff-wireless/files/etc/uci-defaults/24a-fff-wireless
+++ b/src/packages/fff/fff-wireless/files/etc/uci-defaults/24a-fff-wireless
@@ -63,7 +63,7 @@ configWifiDevice() {
 		set wireless.w${freq}configap.network='configap${freq}'
 		set wireless.w${freq}configap.ifname='w${freq}configap'
 		set wireless.w${freq}configap.mode='ap'
-		set wireless.w${freq}configap.ssid='config.franken.freifunk.net'
+		set wireless.w${freq}configap.ssid='config.schleswig-flensburg.freifunk.net'
 		set wireless.w${freq}configap.encryption='none'
 		set wireless.w${freq}configap.hidden='1'
 		set wireless.w${freq}configap.disabled='1'
