diff --git a/src/packages/fff/fff-dhcp/files/etc/uci-defaults/90-fff-dhcp b/src/packages/fff/fff-dhcp/files/etc/uci-defaults/90-fff-dhcp
index 5b59aaf..f0e73a1 100644
--- a/src/packages/fff/fff-dhcp/files/etc/uci-defaults/90-fff-dhcp
+++ b/src/packages/fff/fff-dhcp/files/etc/uci-defaults/90-fff-dhcp
@@ -9,7 +9,7 @@ uci batch >/dev/null <<EOF
   set dhcp.@dnsmasq[-1].localise_queries='0'
   set dhcp.@dnsmasq[-1].rebind_protection='0'
   set dhcp.@dnsmasq[-1].rebind_localhost='1'
-  set dhcp.@dnsmasq[-1].domain='fff.community'
+  set dhcp.@dnsmasq[-1].domain='ffslfl.community'
   set dhcp.@dnsmasq[-1].expandhosts='0'
   set dhcp.@dnsmasq[-1].nonegcache='0'
   set dhcp.@dnsmasq[-1].authoritative='1'
@@ -17,12 +17,9 @@ uci batch >/dev/null <<EOF
   set dhcp.@dnsmasq[-1].leasefile='/tmp/dhcp.leases'
   set dhcp.@dnsmasq[-1].noresolv='1'
   set dhcp.@dnsmasq[-1].localservice='1'
-  add_list dhcp.@dnsmasq[-1].server='10.50.252.11'
-  add_list dhcp.@dnsmasq[-1].server='10.50.252.0'
-  add_list dhcp.@dnsmasq[-1].server='/in-addr.arpa/10.50.252.11'
-  add_list dhcp.@dnsmasq[-1].server='/in-addr.arpa/10.50.252.0'
-  add_list dhcp.@dnsmasq[-1].server='/ip6.arpa/10.50.252.11'
-  add_list dhcp.@dnsmasq[-1].server='/ip6.arpa/10.50.252.0'
+  add_list dhcp.@dnsmasq[-1].server='10.24.32.1'
+  add_list dhcp.@dnsmasq[-1].server='/in-addr.arpa/10.24.32.1'
+  add_list dhcp.@dnsmasq[-1].server='/ip6.arpa/10.24.32.1'

   set dhcp.client=dhcp
   set dhcp.client.interface='client'
